bitrise.yml

```diff
@@ -39,6 +39,7 @@ workflows:
           but the step would use the default value specified in the `step.yml`
           file if you would not specify another value.
         run_if: true
+        is_skippable: true
```

Comment:
I just want to check it, could the step be run locally for testing?

Reply:
Yes. As I see there should not be any problem running it locally.